tests/ResourceFileCreateCommandTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: alex
+ * Date: 18/03/19
+ * Time: 5:31 AM
+ */
+
+namespace CrestApps\CodeGenerator\Tests;
+
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\File;
+
+class ResourceFileCreateCommandTest extends TestCase
+{
+    public function testCreateResourceFileWithBigIntField()
+    {
+        $this->mockOutFileSystem();
+
+        // arguments we're passing in
+        $fieldString = 'name:foo_count;data-type:bigint';
+
+        // now call Artisan
+        Artisan::call('resource-file:create', ['model-name' => 'TestModel', '--fields' => $fieldString]);
+        // Vacuous assertion to give PHPUnit something to do instead of complaining about a risky test
+        $this->assertTrue(true);
+    }
+
+    public function testCreateResourceFileWithBigIntegerField()
+    {
+        $this->mockOutFileSystem();
+
+        // arguments we're passing in
+        $fieldString = 'name:foo_count;data-type:biginteger';
+
+        // now call Artisan
+        Artisan::call('resource-file:create', ['model-name' => 'TestModel', '--fields' => $fieldString]);
+        // Vacuous assertion to give PHPUnit something to do instead of complaining about a risky test
+        $this->assertTrue(true);
+    }
+
+    public function testCreateResourceFileWithMorphedByManyRelation()
+    {
```

Comment:
And another test to help me figure out why including morphedByMany was blowing up.

Hopefully I've given enough examples to make it a lot easier to create regression tests from reasonably well-formed bug reports.